cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -43,45 +46,42 @@ data ShelleyTxCmdError
   | ShelleyTxMissingNetworkId
   | ShelleyTxSocketEnvError !EnvSocketError
   | ShelleyTxReadProtocolParamsError !FilePath !IOException
-  | ShelleyTxReadSignedTxError !ApiError
+  | ShelleyTxReadSignedTxError !OldApi.ApiError
```

Comment:
This is why I went for the `Typed.` qualified import. `Api.` to me is potentially misleading.

Reply:
I think we should fix the import prefixes once we've retired the old api anyway.